Formula/kube-linter.rb

```diff
@@ -0,0 +1,47 @@
+class KubeLinter < Formula
+  desc "Static analysis tool for Kubernetes YAML files and Helm charts"
+  homepage "https://github.com/stackrox/kube-linter"
+  url "https://github.com/stackrox/kube-linter.git",
+    tag:      "0.1.2",
+    revision: "a6df7881ca012d5ea4d88e2e8a73cbe4c382fc13"
+  license "Apache-2.0"
+  head "https://github.com/stackrox/kube-linter.git"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
```

Comment:
```suggestion
```

Reply:
`make` is always present such as `git`?
I removed it and force-pushed.

Reply:
yeah, it always exists.